src/main/java/org/broadinstitute/consent/http/service/UserService.java

```diff
@@ -137,8 +137,12 @@ public User updateDACUserById(Map<String, User> dac, Integer id) throws IllegalA
         if (StringUtils.isEmpty(updatedUser.getDisplayName())) {
             updatedUser.setDisplayName(existingUser.getDisplayName());
         }
+
+        if (Objects.isNull(updatedUser.getInstitutionId())) {
+            updatedUser.setInstitutionId(existingUser.getInstitutionId());
+        }
         try {
-            userDAO.updateUser(updatedUser.getDisplayName(), id, updatedUser.getAdditionalEmail());
+            userDAO.updateUser(updatedUser.getDisplayName(), id, updatedUser.getAdditionalEmail(), updatedUser.getInstitutionId());
```

Comment:
I think having a sanity check that the institution id is a valid one would be helpful somewhere in this method. We can return quickly with a BadRequestException if someone tries to enter an invalid institution as part of the user object.